### PR TITLE
add empty file test cases to utils_test.cc

### PR DESCRIPTION
**Pull request by redwrasse**
Add a couple more test cases of the empty file (`empty.txt`) to `utils_test.cc`:
- reading empty file does not return nullptr
- reading empty file results in 0 lines written to vector

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @terryheo Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @terryheo Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @rascani  Can you please review this PR ? Thank you!

File: tensorflow/lite/tools/evaluation/utils_test.cc [hunk 1]
```diff
@@ -44,20 +44,27 @@ TEST(UtilsTest, StripTrailingSlashesTest) {
...
+  EXPECT_FALSE(ReadFileLines(empty_path, nullptr));
   EXPECT_FALSE(ReadFileLines(wrong_path, &lines));
 }
 
 TEST(UtilsTest, ReadFileCorrectly) {
   std::string file_path(kLabelsPath);
+  std::string empty_path(kEmptyFilePath);
   std::vector<std::string> lines;
-  EXPECT_TRUE(ReadFileLines(file_path, &lines));
+  std::vector<std::string> emptyLines;
```
Comment (rascani) [inline tensorflow/lite/tools/evaluation/utils_test.cc:59]: `emptyLines` should be `empty_lines`

**Review by rascani — APPROVED** [review]

**Pull request merged**